src/software/ai/hl/stp/tactic/shadow_enemy/shadow_enemy_fsm.cpp

```diff
@@ -122,5 +149,21 @@ void ShadowEnemyFSM::stealAndChip(const Update &event)
         TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT,
         TbotsProto::ObstacleAvoidanceMode::AGGRESSIVE,
         TbotsProto::DribblerMode::MAX_FORCE, TbotsProto::BallCollisionType::ALLOW,
-        AutoChipOrKick{AutoChipOrKickMode::AUTOCHIP, YEET_CHIP_DISTANCE_METERS}));
+        AutoChipOrKick{AutoChipOrKickMode::OFF, 0.0}));
+}
+
+void ShadowEnemyFSM::stealAndPull(const Update &event)
+{
+    auto ball_position = event.common.world_ptr->ball().position();
+    auto face_ball_orientation =
+        (ball_position - event.common.robot.position()).orientation();
+    auto pull_to_here =
+        (event.common.robot.position() - ball_position) * 2 + ball_position;
```

Comment:
why do we need both `goAndSteal` AND `stealAndPull`?

Reply:
I also can't really tell where we are trying to move the ball to, I think you should add some document explaining where we're trying to move the ball